android-components/components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TranslationsStateReducer.kt

```diff
@@ -148,7 +148,6 @@ internal object TranslationsStateReducer {
                 TranslationOperation.FETCH_SUPPORTED_LANGUAGES -> {
                     state.copyWithTranslationsState(action.tabId) {
                         it.copy(
-                            supportedLanguages = null,
```

Comment:
Good thing to mention! 

>another interpretation here would be that supportedLanguages is already null to begin with (since this is a global state now and we would only ever fetch the supported languages once)
* I would expect it to usually be null if we end up here
* If it wasn't null, I'd wonder why we were doing this operation and hesitant to null it out (a successful operation happened somewhere, so probably should be okay with the result